eng/testing/tests.mobile.targets

```diff
@@ -125,6 +125,20 @@
       <AssemblySearchPaths Include="$(PublishDir)"/>
       <WasmFilesToIncludeInFileSystem Include="@(ContentWithTargetPath)" />
       <WasmFilesToIncludeInFileSystem Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.BuildReference)' == 'true'" />
+      <ExtraAssemblies Include="$(PublishDir)$(AssemblyName).dll" />
+      <!-- we need to preserve these facades for BinaryFormatter tests -->
+      <ExtraAssemblies Include="$(PublishDir)mscorlib.dll" />
```

Comment:
Why not publish trimmed and remove the custom trimming we do in the AppBuilder?

Reply:
that's the plan but this was quicker for now.